### ISSUE TITLE
Cursor being repositioned incorrectly

### ISSUE THREAD

**Issue by jameshibbard** [bahamas10/node-clear#4]
Module isn't working for me on bash on Linux Mint.

```
bash --version
$ GNU bash, version 4.3.11(1)-release (x86_64-pc-linux-gnu)
```

I did:

```
npm install clear
node
var clear = require("clear");
clear();
```

I see:

![image](https://cloud.githubusercontent.com/assets/1940994/16454651/cc73308a-3e11-11e6-990b-0a2a152dcd36.png)

Notice the weird character before `undefined`?

I think this has something to do with the character sequence you are using to reset the cursor position:

In `clear/index.js` line 5 we have: 

```
process.stdout.write('\033[0f');
```

According to [this page](http://www.tldp.org/HOWTO/Bash-Prompt-HOWTO/x361.html):

> Position the Cursor:   \033[<L>;<C>H      Or   \033[<L>;<C>f   puts the cursor at line L and column C.

And sure enough, changing it to:

```
process.stdout.write('\033[0;0f');
```

Makes things work as expected:

![image](https://cloud.githubusercontent.com/assets/1940994/16454826/805b0eec-3e12-11e6-90f8-c7e13e1f9b9a.png)

**Comment by lequer** [comment]
Same on Ubuntu 16.04, bash version 4.3.46
`process.stdout.write('\033[0;0f');`
work as expected.

**Comment by bahamas10** [comment]
Thanks for the detailed report - sorry I didn't see this sooner.  I just published clear@0.1.0 now.  I don't have an Ubuntu machine to test on but if you could test again that would be much appreciated.  If it still fails I can look into modifying it to use the sequence you pasted above.

**Comment by jameshibbard** [comment]
Hi, I can confirm that this works on the latest version of Linux Mint (18.3). Thanks for the fix.
